### PR TITLE
describe the state field

### PR DESCRIPTION
**Pull request by zeroheure**
I've added both field content and help to ease understand the possibilities (draft and sellable have different meaning than In Development and Normal

### CODE REVIEW

**Review by bodedra — APPROVED** [review]
LGTM :+1:

File: product_state/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1 +1,6 @@
-This module introduces the state field on product template and allows simple product life cycle.
+This module introduces the state field on product template and allows simple product life cycle :
```
Comment (pedrobaeza) [inline product_state/readme/DESCRIPTION.rst]: Why putting an space between last word and `:`?

Comment (zeroheure) [reply product_state/readme/DESCRIPTION.rst]: Typographic habit. Removed.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**